functable.c

```diff
@@ -249,103 +216,103 @@ static void init_functable(void) {
 }
 
 /* stub functions */
-static uint32_t update_hash_stub(deflate_state* const s, uint32_t h, uint32_t val) {
+static uint32_t adler32_stub(uint32_t adler, const uint8_t* buf, size_t len) {
     init_functable();
-    return functable.update_hash(s, h, val);
+    return functable.adler32(adler, buf, len);
 }
 
-static void insert_string_stub(deflate_state* const s, uint32_t str, uint32_t count) {
+static uint32_t adler32_fold_copy_stub(uint32_t adler, uint8_t* dst, const uint8_t* src, size_t len) {
     init_functable();
-    functable.insert_string(s, str, count);
+    return functable.adler32_fold_copy(adler, dst, src, len);
 }
 
-static Pos quick_insert_string_stub(deflate_state* const s, const uint32_t str) {
+static uint8_t* chunkmemset_safe_stub(uint8_t* out, unsigned dist, unsigned len, unsigned left) {
     init_functable();
```

Comment:
These Codecov warnings were direct result of moving the functable initialization to single function instead of initializing each functable member in own stub function. The end result is that we have (stub) functions that are never executed by test suite so can possibly (in future) contain bad or dead code.